circle.yml

```diff
@@ -67,6 +67,11 @@ deployment:
     owner: palantir
     commands:
       - ./scripts/circle-ci/publish-github-page.sh
+  publish-snapshots:
+    branch: /.*/
+    owner: atlasdb
```

Comment:
This is the organisation under which the build is running when deploying. Which will be `atlasdb` in the case when we want to publish.